testproject/tests.py

```diff
@@ -72,6 +72,12 @@ def test_find_static_sass(self):
             )
             in files
         )
+        self.assertTrue(
+            os.path.join(
+                THIS_DIR, "app3", "static", "app3", "sass", "test.sass"
+            )
+            in files
+        )
```

Comment:
Can you also add a test which compiles the `test.sass`, and verifies the outputted CSS? (see `test_cli` and `assert_output` methods)